src/autoevals/string.py

```diff
@@ -1,19 +1,27 @@
 from Levenshtein import distance
 
-from .base import Evaluator
+from autoevals.base import Evaluation
+
+from .base import Evaluation, Evaluator
 
 
 class LevenshteinEvaluator(Evaluator):
-    async def _run_eval_sync(self, output, expected=None, **kwargs):
+    async def _run_eval_async(self, output, expected=None, **kwargs):
```

Comment:
i think you can just omit this (it should get inherited from the default implementation)

Reply:
Remooved: https://github.com/braintrustdata/autoevals/pull/3/commits/ede368311c534574c3ff8d2e4f4099429a03608a#diff-2a70ff6c478aeed7d624e254df20fad6598c77fc4a6aa04def2d2ad1a9e84658L9